packages/authentication/src/definitions.ts

```diff
@@ -667,6 +673,38 @@ export interface GetIdTokenResult {
   token: string;
 }
 
+export interface GetIdTokenInfo extends GetIdTokenResult {
```

Comment:
```suggestion
export interface GetIdTokenInfo {
```

Just add a separate `token` property.